index.html

```diff
@@ -1422,11 +1462,7 @@ <h3>Types</h3>
 All <a>credentials</a>, <a>presentations</a>, and encapsulated objects MUST
 specify, or be associated with, additional more narrow <a>types</a> (like
 <code>UniversityDegreeCredential</code>, for example) so software systems can
-process this additional information. The specification of these additional more
-narrow types could be done in the object within which the additional information
-is found, rather than at a higher level such as the <code>type</code>
-<a>property</a> of the <a>credential</a> or <a>presentation</a>. It is also
```

Comment:
I suggest we just strike "It is also possible for the additional more narrow types to be implicit." and call it a day. The CR specification already said "or be associated with" ... and there are MANY ways to do that, not just the ones outlined in this PR.

Reply:
I support Manu's proposal

Reply:
I support it as well